stackit/stackit_core.py

```diff
@@ -208,6 +213,13 @@ def main(config, search, stderr, tag, verbose, version):
         _search(config)
     elif version:
         click.echo("Version {VERSION_NUM}".format(**globals()))
+    else:
+        click.echo(
+            click.style(
+                "No argument provided, use --help for help",
+                fg="red",
+                err=True))
+        sys.exit(1)
```

Comment:
I also add this error message when `stackit` is launched without arguments, let me know I you prefer another one